db/tidb/bank_qos.go

```diff
@@ -13,15 +13,36 @@ import (
 )
 
 type rtoMetrics struct {
+	name             string
 	nemesis          *core.NemesisGeneratorRecord
+	totalMeasureTime time.Duration
+	baseline         float64
 	p80              time.Duration
 	p90              time.Duration
 	p99              time.Duration
-	totalMeasureTime time.Duration
 }
 
-type bankServiceQualityChecker struct {
-	profFile string
+func (m *rtoMetrics) Record(filePath string) error {
+	os.MkdirAll(path.Dir(""), 0755)
+	f, err := os.OpenFile(filePath, os.O_CREATE|os.O_APPEND|os.O_WRONLY, 0644)
```

Comment:
does it need to be `O_SYNC`?

Reply:
It's more safe if so but we don't need that guarantee here.